DOCUMENTATION.md

```diff
@@ -1686,6 +1686,10 @@ The following UI controls are available:
   * [VRTK_Lever](#vrtk_lever)
   * [VRTK_Slider](#vrtk_slider)
 
+All controls emit the following events:
```

Comment:
As this overview page isn't on the readme.io site, it may be better to have the `### Class Events` section on each of the controls docs (plus the `#### Event Payload` they emit)

Then if people jump to a specific control they can read the events for that specific control (even though it's just a copy and paste)